PubSub/src/V1/Gapic/SchemaServiceGapicClient.php

```diff
@@ -466,13 +465,16 @@ public function deleteSchema($name, array $optionalArgs = [])
      *
      * @throws ApiException if the remote call fails
      */
-    public function deleteSchemaRevision($name, $revisionId, array $optionalArgs = [])
+    public function deleteSchemaRevision($name, array $optionalArgs = [])
```

Comment:
This is a breaking change and will require us to add an exception in the generator